requirements/base.txt

```diff
@@ -9,4 +9,5 @@ six>=1.10
 tqdm>=4.44
 urllib3 >= 1.25.3
 pydantic >= 1.10.5, < 2
-aenum >= 3.1.11
\ No newline at end of file
+aenum >= 3.1.11
+packaging
```

Comment:
Didn't specify minimal version as the package should already be installed in 99% of the cases as it is a dependency of many other packages. The functionality we require from the package is also available in very old versions. I wouldn't expect any issues here.